DevoxxClientMobile/src/main/java/com/devoxx/views/SponsorsPresenter.java

```diff
@@ -76,13 +83,14 @@ public void initialize() {
 
         sponsorListView.setCellFactory(p -> new SponsorCell());
         sponsorListView.setHeaderCellFactory(p -> new SponsorHeaderCell());
+        sponsorListView.setComparator((s1, s2) -> s1.getName().compareTo(s2.getName()));
 
-        sponsorListView.selectedItemProperty().addListener((observable, oldValue, newValue) -> {
-            if (newValue != null) {
-                DevoxxView.SPONSOR.switchView().ifPresent(presenter ->
-                        ((SponsorPresenter)presenter).setSponsor(newValue.getName(), newValue.getSlug()));
-            }
-        });
+//        sponsorListView.selectedItemProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
remove commented code? or is it supposed to come back somehow?

Reply:
SessionsPresenter also has the commented code. It can be used as a quick reference in the future to fix the bug.